README.md

```diff
@@ -1,5 +1,8 @@
 # [Cozy](https://cozy.io) Desktop
 
+[![Build Status](https://travis-ci.org/cozy-labs/cozy-desktop.png)](https://travis-ci.org/cozy-labs/cozy-desktop)
```

Comment:
yes but it's always on the top because it's a priority to show it (for lot of people and for me) ;)